.travis.yml

```diff
@@ -33,4 +33,4 @@ before_script:
 
 script:
   - npm run travis:$JOB_PART
-  - commitlint-travis
\ No newline at end of file
+  - commitlint-travis
```

Comment:
Please keep newline at EOF. :smiley_cat:

Reply:
I did not remove it , I added it

Reply:
Oh yes. Thanks. :heart:

Reply:
Oh yes. Thanks. :heart: